.github/workflows/tests-k8s.yaml

```diff
@@ -1,4 +1,5 @@
-name: Tests K8s
+name: Charm Analysis
```

Comment:
Should we change the name of the file to `charm-analysis.yaml` and keep things consistent?

Reply:
as you wish